precompiles/utils/src/solidity/codec/bytes.rs

```diff
@@ -95,6 +95,8 @@ impl<K: Kind, S: Get<u32>> Codec for BoundedBytesString<K, S> {
 			.try_into()
 			.map_err(|_| RevertReason::value_is_too_large("length"))?;
 
+		reader.move_cursor(32)?;
```

Comment:
I also think this is wrong, but I can't remember why -- I just remember finding that I didn't actually need to fix anything :)

Reply:
If that is the case please add documentation as to why it was thought wrong initially and why it is not so we avoiding doing this again :)

Reply:
I opened a Jira to do exactly this :)

Reply:
So, [the original issue](https://opslayer.atlassian.net/browse/MOON-2279) on which this PR is based is invalid and should be closed, right?

Reply:
Yes, that's right. Feel free to close this PR.